server/src/service/AccountsService.js

```diff
@@ -10,20 +10,20 @@ async function createNewAccount(receivedData) {
             const data = await accountsDao.createNewAccount({
                 username: receivedData.username,
                 password: receivedData.password, // encrypt this
-                isAdmin: receivedData.isAdmin,
-                
-            })
+                isAdmin: receivedData.isAdmin
+            });
+            return data;
         }
+        return 'username already exists';
     }
-
-    
+    return null;    
 }
 
 
 async function accountDoesExist(username) {
```

Comment:
Reminder:
This function seems excessive for just adding a new account.  In the future, look into using the DynamoDB `ConditionExpression` `attribute_not_exists` for the `PutCommand`.

Reply:
This is good advice, I will look into fixing this in the future if time permits but functionality is working as it needs to and we need to start building out the rest of the features. I'll make a note and circle back to it.